MpsAllocatorSample/Program.cs

```diff
@@ -234,9 +228,7 @@ static async Task PrintServerStateInVM()
             var req = new PlayFab.MultiplayerModels.ListMultiplayerServersRequest();
             string buildID = ReadBuildIDFromInput();
             var regions = await GetRegions(buildID);
```

Comment:
>await GetRegions(buildID); [](http://example.com/codeflow?start=26&length=26)

Nit: 'regions' is only used to be passed as parameter to the other function. Maybe we can just consolidate it inside 'ReadRegionFromInput'.